governance-and-automation/automated-workflows/inactive-workspaces-admin.md

```diff
@@ -0,0 +1,79 @@
+  * **Select the actions that owners can perform when they receive a task (9)**:
+    * **Keep**
+      * This action is visible by default, and if the workspace is kept, it won't be marked as inactive for the period defined in the first step
+    * **Archive**
+      * If this action is selected and the workspace is archived, it can still be restored later
+    * **Delete**
+      * If this action is selected, and a workspace is deleted, it can be restored from the recycle bin within 93 days, after which it is deleted for good
+  * **Enter the number of total work days reviewers have to complete the task (10)**:
+    * **Click the checkbox if you want a reminder sent (11)** 3 work days before the due date 
+  * **Select what Syskit Point should do if owners do not resolve this task until due date (12)**:
+    * Perform no action
+    * Automatically Archive
+    * Automatically Delete
+* Click **Finish (13)** once you are done with the policy configuration   
+
+![Edit Policy Dialog - Step 1](../../.gitbook/assets/inactive-workspaces-admin-edit-step-1.png)
+
+![Edit Policy Dialog - Step 2](../../.gitbook/assets/inactive-workspaces-admin-edit-step-2.png)
+
+![Edit Policy Dialog - Step 3](../../.gitbook/assets/inactive-workspaces-admin-edit-step-3.png)
+
+**Syskit Point will start the task delegation workflow based on your configuration when an inactive workspace is detected**. 
+
+For details on how collaborators can [**resolve Inactive Workspaces tasks**, navigate to the following article](../../point-collaborators/resolve-governance-tasks/inactive-workspaces.md).
+
+## Detection for Inactive Workspaces
+
+To decide whether something is inactive in your Microsoft 365 environment, we collect activity data for **SharePoint, Exchange, Yammer, and Microsoft Teams**. If any kind of activity is detected, the workspace will not be declared inactive. The amount of time that needs to pass for a workspace to be considered inactive is determined by administrators when creating or editing the Inactive Workspaces policy. 
+
+The following type of activity is checked to determine if a workspace is inactive:
+
+* **SharePoint activity** - collected via Microsoft 365 audit logs. **The last activity is based on the latest date when a file was viewed, edited, or downloaded**.
+* **Exchange activity** - collected via Microsoft 365 Groups activity report available in the Microsoft 365 admin center. **The last activity is based on the latest date when an email was delivered to a group**.
+* **Yammer activity** - the collection is also accomplished through the Microsoft 365 admin center usage report for Yammer groups activity. **The last activity is based on the latest date a message was read, posted, or liked by the group**.
+* **Microsoft Teams** - activity is collected using protected APIs in Microsoft Graph. **The last activity is based on the latest date when a message was posted or replied to**.
+
+
+The following conditions also have to be met:
```

Comment:
ne bih ovo nazvao conditions, to su pravila / logika kako radi Point po pitanju detekcije